### PR TITLE
[DOCS]chore: add style fixes

### PR DESCRIPTION
**Pull request by sstraatemans**

### CODE REVIEW

File: packages/libs/react-ui/src/components/Notification/Notification.css.ts [hunk 1]
```diff
@@ -59,6 +59,7 @@ export const contentClass = style([
     display: 'flex',
     flexDirection: 'column',
     gap: '$xs',
+    width: '100%',
```
Comment (sstraatemans) [inline packages/libs/react-ui/src/components/Notification/Notification.css.ts:62]: fixes an issue with the size of the Icon. for some reason with a long text, the icon gets super small

image

File: packages/apps/docs/src/components/Layout/components/TreeMenu/styles.css.ts [hunk 2]
```diff
@@ -9,6 +9,7 @@ export const treeListClass = style([
     padding: 0,
   }),
   {
+    overflowX: 'hidden',
```
Comment (sstraatemans) [inline packages/apps/docs/src/components/Layout/components/TreeMenu/styles.css.ts:12]: removes the small horizontal scroll error in the left sidemenu

File: packages/apps/docs/src/components/Markdown/Code/style.css.ts [hunk 3]
```diff
@@ -19,7 +19,7 @@ export const codeWrapper = style([
     fontFamily: '$mono',
     lineHeight: '$lg',
     marginX: '$5',
-    marginY: 0,
+    marginY: '$10',
```
Comment (sstraatemans) [inline packages/apps/docs/src/components/Markdown/Code/style.css.ts:22]: more margin between codeblocks and rest of content on a page

File: packages/apps/docs/src/components/Markdown/UnorderedList/styles.css.ts [hunk 4]
```diff
@@ -1,23 +1,12 @@
...
 export const ulListClass = style([
   sprinkles({
     marginY: '$5',
     marginX: 0,
     position: 'relative',
+    color: '$neutral4',
   }),
 ]);
-
-globalStyle(`.${ulListClassName}  ul`, {
```
Comment (sstraatemans) [inline packages/apps/docs/src/components/Markdown/UnorderedList/styles.css.ts:15]: this didnt do anything

File: packages/apps/docs/src/components/Markdown/styles.css.ts [hunk 5]
```diff
@@ -1,10 +1,34 @@
...
 
 export const wrapperClass = style([
   sprinkles({
     marginY: '$5',
     marginX: 0,
   }),
+  {},
 ]);
+
+globalStyle(
```
Comment (sstraatemans) [inline packages/apps/docs/src/components/Markdown/styles.css.ts:16]: for a MDnotifications paragraphs should not have margins

**Review by realdreamer — APPROVED** [review]
LGTM 🔥

File: packages/apps/docs/src/components/BottomPageSection/styles.css.ts [hunk 6]
```diff
@@ -4,6 +4,7 @@ import { style } from '@vanilla-extract/css';
 
 export const bottomWrapperClass = style([
   sprinkles({
+    marginTop: '$40',
```
[the remark this answers is not in the document]
Comment (sstraatemans) [reply packages/apps/docs/src/components/BottomPageSection/styles.css.ts:7]: isa wanted a lot of space :)

File: packages/apps/docs/src/components/BottomPageSection/BottomPageSection.tsx [hunk 7]
```diff
@@ -41,7 +41,7 @@ export const BottomPageSection: FC<IProps> = ({
   };
 
   return (
-    <div className={classes}>
```
Comment (eileenmguo) [inline packages/apps/docs/src/components/BottomPageSection/BottomPageSection.tsx:44]: can you use the react-ui footer for this?

**Review by eileenmguo — APPROVED** [review]

File: packages/apps/docs/src/components/BottomPageSection/BottomPageSection.tsx (hunk 7 above)
Comment (sstraatemans) [reply packages/apps/docs/src/components/BottomPageSection/BottomPageSection.tsx:44]: diff style.

**Pull request merged**